src/table/__integ__/performance-marks.test.ts

```diff
@@ -27,29 +27,19 @@ function setupTest(
 
 describe('Table', () => {
   test(
-    'Emits a mark only for visible tables',
+    'Emits a mark only for visible tables which are completed loaded',
```

Comment:
Let's split out this change (waiting for tables to finish loading) into a separate pull request, since it's not directly related to re-running the marking effect when the event is emitted

Reply:
Just for tables or buttons as well?